package.json

```diff
@@ -60,46 +59,40 @@
     "react-dom": "^15.0.0 || ^16.0.0"
   },
   "dependencies": {
-    "@babel/polyfill": "^7.2.5",
-    "@babel/runtime": "^7.2.0",
-    "classnames": "^2.2.6",
+    "@babel/polyfill": "^7.4.4",
+    "@babel/runtime": "^7.4.4",
     "cornerstone-core": "^2.2.8",
-    "cornerstone-math": "^0.1.8",
-    "cornerstone-tools": "^3.2.4",
+    "cornerstone-tools": "^3.7.2",
     "cornerstone-wado-image-loader": "^2.2.3",
-    "dcmjs": "^0.3.8",
+    "dcmjs": "^0.4.7",
     "dicom-parser": "^1.8.3",
-    "dicomweb-client": "^0.4.2",
+    "eslint-config-react-app": "^4.0.0",
     "hammerjs": "^2.0.8",
-    "lodash.isequal": "4.5.0",
+    "husky": "1.3.x",
     "moment": "^2.24.0",
-    "ohif-core": "0.4.2",
+    "ohif-core": "0.5.0",
     "ohif-cornerstone-extension": "^0.0.28",
     "ohif-dicom-html-extension": "^0.0.2",
     "ohif-dicom-microscopy-extension": "^0.0.5",
     "ohif-dicom-pdf-extension": "^0.0.6",
     "ohif-vtk-extension": "^0.0.1",
```

Comment:
Can we lock all pre-v1.0.0 dependencies to an exact version?
And all post-v1.0.0 dependencies to `Major.Minor.x` (where we'll only update to latest patch)?